example_11/CMakeLists.txt

```diff
@@ -16,6 +16,14 @@ set(THIS_PACKAGE_INCLUDE_DEPENDS
   rclcpp_lifecycle
 )
 
+# Specify the required version of ros2_control
+find_package(controller_manager 4.0.0)
```

Comment:
```suggestion
find_package(controller_manager 4.0.0 REQUIRED)
```

Reply:
with `required` fails immediately and the error message is not printed any more. I think this might be useful for ROS beginners (which might be the major audience of the demo repositories).

Reply:
if you now a way to provide a custom error message with `REQUIRED`, this would be ideal.

Reply:
You are right!. Thanks for the explanation